cap-ci/README.md

```diff
@@ -11,11 +11,10 @@ tests KubeCF on Kubernetes distributions CaaSP4, GKE, EKS, AKS:
 
 # Deploying the pipeline
 
-    $ ./create_cap-release_pipeline.sh
+    $ ./create_cap-release_pipeline.sh <concourse-target> <pipeline-name>
 
-If you want your own pipeline: `PIPELINE=yourname ./create_cap-release_pipeline.sh`
-See `BACKEND`, `OPTIONS`, `EIRINI` to disable some features on deployment.
-The new `yourname` pipeline will make use of kuceconfigs uploaded to EKCP
+Configure all the required options in `config.yaml`.
+The new `<pipeline-name>` pipeline will make use of kuceconfigs uploaded to EKCP
```

Comment:
wow this is outdates. kubeconfigs are uploaded to cf-ci-pools

Reply:
Actually its already here https://github.com/SUSE/cap/tree/bisingh/ci-cleanup/cap-ci#k8s-cluster-management 
all we need to do is, remove ekcp statements

Reply:
let's handle it in a separate PR.